### ISSUE TITLE
请问如何才能使用Large-Margin softmax loss呢？

### ISSUE THREAD

**Issue by yeyupiaoling** [PaddlePaddle/Paddle#16564]
# 环境
 - PaddlePaddle 版本随意
 - Ubuntu 16.04
 - Python 3.5

# 问题
查阅了很多资料，多说人脸识别分类这种大规模的分类使用传统的softmax，而是使用Large-Margin softmax loss（L-softmax loss），这个是怎么个使用法呢？

或者是否有更好的函数呢？

**Comment by chengduoZH** [comment]
你需要说明一下Large-Margin softmax loss（L-softmax loss）的公式是什么？

**Comment by yeyupiaoling** [comment]
@chengduoZH 我以为paddlepaddle内部实现了呢？
我以为是跟使用softmax，然后在使用 cross_entropy 的使用方式一样呢。比如在最后一层的全连接层使用的激活函数是`L-softmax`，然后损失函数使用cross entropy。我是不是想得太简单了。

公式如下：
![image](https://user-images.githubusercontent.com/26297768/55278337-ca191100-5345-11e9-8731-29bd4e4dc06a.png)

**Comment by chengduoZH** [comment]
目前Paddle没有相关Op的实现，您可以先根据公式推导，用其他Op构建一个这样的公式。

**Comment by yeyupiaoling** [comment]
好的，谢谢。
